{"name": "test_case09", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_autouse.Testfixture object at 0x00000217A600FF60>\n\n    def test_case09(self):\n        print('=========test_case09 ======')\n>       assert False\nE       assert False\n\ntest_autouse.py:15: AssertionError"}, "start": 1647685262209, "stop": 1647685262209, "uuid": "847dc7d7-ec08-4a98-ab82-c4679f86d124", "historyId": "a450ab6ca5ca64992d42cd44f6e52a1b", "testCaseId": "3befa9224f89d250df19172d45e78d42", "fullName": "test_autouse.Testfixture#test_case09", "labels": [{"name": "suite", "value": "test_autouse"}, {"name": "subSuite", "value": "Testfixture"}, {"name": "host", "value": "WIN-N5AGDL06G55"}, {"name": "thread", "value": "41080-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_autouse"}]}